components/scream/src/physics/shoc/shoc_iso_c.f90

```diff
@@ -497,4 +497,14 @@ subroutine linear_interp_c(x1,x2,y1,y2,km1,km2,ncol,minthresh) bind (C)
 
   end subroutine linear_interp_c  
 
+ subroutine shoc_diag_second_moments_ubycond_c(shcol, thl, qw, wthl, wqw, qwthl, uw, vw, wtke) bind(C)
+   use shoc, only: diag_second_moments_ubycond
+
+   ! argmens
```

Comment:
Can change to "arguments" or omit this comment. I think it's clear that what follows are arguments to the subroutine.